app/scripts/controllers/preferences.js

```diff
@@ -621,6 +624,18 @@ class PreferencesController {
     return Promise.resolve(true)
   }
 
+  //
+  // Brave
+  //
+
+  /**
+   * Sets the {@code batTokenAdded} state to {@code true}, indicating that the default addition of the BAT token has completed
+   */
+  setBatTokenAdded () {
+    this.store.updateState({ batTokenAdded: true })
+    return Promise.resolve(true)
+  }
```

Comment:
Could the changes for this be made into a subclass and then have this path resolve to the Brave export instead?